src/javascript/ContentEditor.context.js

```diff
@@ -98,7 +101,8 @@ export const withContentEditorDataContextProvider = (formQuery, formDataAdapter)
             formQueryParams,
             nodeTypeName,
             nodeTypeDisplayName,
-            refetchFormData
+            refetchFormData,
+            navigatorLocale
```

Comment:
I would have name it something like browserLang, and put this variable next to lang and uiLang for consistency.

Reply:
Done.